core/tests/protractor_utils/forms.js

```diff
@@ -461,11 +486,11 @@ var expectRichText = function(elem) {
         // applying .getText() while the RichTextChecker is running would be
         // asynchronous and so not allow us to update the textPointer
         // synchronously.
-        return await entry.getText();
+        return await action.getText('Entry', entry);
       });
     // We re-derive the array of elements as we need it too.
     var arrayOfElements = elem.all(by.xpath(XPATH_SELECTOR));
-    var fullText = await elem.getText();
+    var fullText = await action.getText('Elem', elem);
```

Comment:
Do we know what `elem` is? Can we get a more descriptive name for the first argument?

Reply:
i think it's  related to field text so i named it.
done!